eks-without-vpc/outputs.ft

```diff
@@ -0,0 +1,69 @@
+#
```

Comment:
Filename extension should be `tf` instead of `ft`

Reply:
done